profile/README.md

```diff
@@ -57,6 +57,19 @@ You can learn more about Prisma and its rich ecosystem and community with these
 
 </details>
 
+## Badges
+
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io) [![Made with Prisma](http://made-with.prisma.io/indigo.svg)](https://prisma.io)
+
+Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your readme or website.
+
+```
```

Comment:
we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

Reply:
Thanks for the suggestion: I'll add the same comment as on the other repo. The syntax highlighting didn't show up for me as any difference on presskit, so I'll just merge this in. Very happy to add this later on if it turns out maybe I missed something.